### DISCUSSION TITLE
请问各位有人能到空洞里帮我拿个东西吗，在线等，急

### DISCUSSION THREAD

**Discussion by Lanvika** [share121/inter-knot#464] [其他]
我邦布不知为何跑空洞里去了，最新款的鲨牙布，就在埃尔格街的中型空洞里，如有找到，必有重谢
![AAA邦布立绘-鲨牙布](https://github.com/user-attachments/assets/12d49853-aaf7-4454-baa0-23a8403beb49)

**Comment by Lanvika** [comment]
好贵好贵的，整整八十八万丁尼啊

**Comment by AKT505** [comment]
兄弟……

**Comment by KUNKUN4** [comment]
给我20万菲林就帮你

**Comment by XiaoFffeng** [comment]
建议直接@法厄同

**Comment by ncateuc** [comment]
2万菲林，这个委托我接了。

**Comment by ncateuc** [comment]
2万菲林，这个委托我接了。
